app/controllers/vacancies_controller.rb

```diff
@@ -126,4 +126,8 @@ def specific_phases?
   def set_headers
     response.set_header('X-Robots-Tag', 'noarchive')
   end
+
+  def smoke_test?
+    cookies[:smoke_test] != nil
```

Comment:
Thought of potentially a better way to handle this with just:

```
cookies[:smoke_test].present?
```

Reply:
I was looking for this solution but it throws undefined method error